next/package.json

```diff
@@ -22,29 +22,33 @@
+    "update-rjsf": "npm install @rjsf/core@latest @rjsf/utils@latest @rjsf/validator-ajv8@latest ajv@latest ajv-formats@latest --save-exact",
+    "update-markdown": "npm install remark-directive@latest remark-directive-rehype@latest remark-gfm@latest remark-supersub@latest rehype-raw@latest rehype-sanitize@latest",
+    "update-next": "npm install next@latest react@latest react-dom@latest sharp@latest --save-exact && npm install @types/react@latest @types/react-dom@latest eslint-config-next@latest typescript@latest @next/bundle-analyzer --save-dev --save-exact",
+    "update-amplify": "npm install aws-amplify@latest @aws-amplify/adapter-nextjs@latest --save-exact",
+    "update-tailwind": "npm install tailwind-merge@latest tailwind-scrollbar-hide@latest --save-exact && npm install tailwindcss@latest autoprefixer@latest postcss@latest prettier-plugin-tailwindcss@latest eslint-plugin-tailwindcss@latest --save-dev --save-exact",
+    "update-i18next": "npm install i18next@latest next-i18next@latest react-i18next@latest --save-exact",
+    "update-prettier": "npm install prettier@latest eslint-config-prettier@latest --save-dev --save-exact",
+    "update-eslint": "npm install eslint@latest @typescript-eslint/eslint-plugin@latest @typescript-eslint/parser@latest eslint-config-adjunct@latest eslint-config-airbnb@latest eslint-config-airbnb-typescript@latest eslint-config-auto@latest eslint-config-next@latest eslint-config-prettier@latest eslint-plugin-array-func@latest eslint-plugin-const-case@latest eslint-plugin-eslint-comments@latest eslint-plugin-html@latest eslint-plugin-import@latest eslint-plugin-jest@latest eslint-plugin-jest-async@latest eslint-plugin-jest-dom@latest eslint-plugin-json@latest eslint-plugin-lodash@latest eslint-plugin-lodash-fp@latest eslint-plugin-markdown@latest eslint-plugin-no-constructor-bind@latest eslint-plugin-no-secrets@latest eslint-plugin-no-unsanitized@latest eslint-plugin-no-use-extend-native@latest eslint-plugin-optimize-regex@latest eslint-plugin-pii@latest eslint-plugin-prettier@latest eslint-plugin-promise@latest eslint-plugin-react-hooks@latest eslint-plugin-scanjs-rules@latest eslint-plugin-security@latest eslint-plugin-simple-import-sort@latest eslint-plugin-sonarjs@latest eslint-plugin-switch-case@latest eslint-plugin-tailwindcss@latest eslint-plugin-testing-library@latest eslint-plugin-unicorn@latest eslint-plugin-xss@latest @tanstack/eslint-plugin-query@latest @typescript-eslint/eslint-plugin@latest @typescript-eslint/parser@latest --save-dev --save-exact",
+    "update-graphql": "npm install @graphql-codegen/cli@latest @graphql-codegen/client-preset@latest @graphql-codegen/typescript-graphql-request@latest --save-dev --save-exact && npm install graphql@latest graphql-request@latest graphql-tag@latest --save-exact"
+  },
+  "overrides": {
```

Comment:
Why are these overrides needed?

Can we get rid of them? If not, can you explain why in `package.json`. It's important to document these decisions because it's not clear for other people why are they here.